### PR TITLE
Removing duplicate prediction call in ProphetModel, replacing Githun with Github

### PR DESCRIPTION
**Pull request by Axemen**
## Kats

Thank you for contributing! Please take a moment to review our [**contributing guidelines**](https://github.com/facebookresearch/Kats/blob/master/CONTRIBUTING.md)
to make the process easy and effective for everyone involved.

- [x] Fork the repo and create your branch from master.
- [x] If you've added code that should be tested, add tests.
- [x] If you've changed APIs, update the documentation.
- [x] Ensure the test suite passes.
- [x] Make sure your code lints.
- [x] If you haven't already, complete the Contributor License Agreement ("CLA").

Contributor License Agreement ("CLA")
In order to accept your pull request, we need you to submit a CLA. You only need to do this once to work on any of Facebook's open source projects.

Complete your CLA here: https://code.facebook.com/cla

`Ver: Kats v1.02`

### CODE REVIEW

File: kats/models/prophet.py [hunk 1]
```diff
@@ -320,12 +320,10 @@ def predict(self, steps, include_history=False, **kwargs) -> pd.DataFrame:
             if self.params.floor is not None:
                 future["floor"] = self.params.floor
 
-        fcst = self.model.predict(future).tail(steps)
+        fcst = self.model.predict(future)
         if raw:
-            return fcst
+            return fcst.tail()
```
Comment (jeffhandl) [inline kats/models/prophet.py:325]: Shouldn't this be `fcst.tail(steps)`?

**Review by jeffhandl — CHANGES_REQUESTED** [review]
One change requested - the rest looks good!

File: kats/models/prophet.py (hunk 1 above)
Comment (Axemen) [reply kats/models/prophet.py:325]: Yes it should be!

**Comment by Axemen** [comment]
@jeffhandl Just made the change requested in commit 
b164f50

**Pull request closed without merging**